pkg/synthetictests/allowedalerts/allow_on_revision_change.go

```diff
@@ -25,7 +25,10 @@ var _ AlertTestAllowanceCalculator = &etcdRevisionChangeAllowance{}
 func NewAllowedWhenEtcdRevisionChange(ctx context.Context, kubeClient kubernetes.Interface, duration time.Duration) (*etcdRevisionChangeAllowance, error) {
 	numberOfRevisions, err := GetEstimatedNumberOfRevisionsForEtcdOperator(ctx, kubeClient, duration)
 	if err != nil {
-		return nil, err
+		// instead of panicking when there no etcd operator (e.g. microshift), just estimate the number of revisions to be 1
```

Comment:
Please check for the specific error that you're getting.  I bet you're getting a 404 (apierrors.IsNotFound).

Reply:
Is the server running/reachable?

`GetEstimatedNumberOfRevisionsForEtcdOperator` starts with: https://github.com/openshift/origin/blob/3cbe9f27e91966902521bb54396a4573e9138698/pkg/synthetictests/allowedalerts/allow_on_revision_change.go#L55-L59

Does `openshift-etcd` namespace exist in the MicroShift cluster?

Reply:
No, there is no etcd server running/reachable, and there is no openshift-etcd namespace in a microshift cluster.

Reply:
syscall.ECONNREFUSED isn't being returned anymore in my latest tests, but apierrors.IsNotFound is being returned, so I added a case for that. 

GetEstimatedNumberOfRevisionsForEtcdOperator doesn't seem to be returning errors anymore (it returns an empty list of ConfigMaps because the openshift-etcd NS doesn't exist), so I removed the error handling there.